scripts/helpers.js

```diff
@@ -154,3 +154,8 @@ hexo.extend.helper.register('disqus_lang', function() {
 
   return data.disqus_lang || lang;
 });
+
+hexo.extend.helper.register('hexo_version', function() {
+  return this.version;
```

Comment:
I got undefined when using `this.version` while correctly when using `hexo.version`. It should be `this.env.version` or `hexo.version` I think.
https://github.com/hexojs/hexo/blob/master/lib/hexo/index.js#L55

Reply:
Sorry, I just made a quick change but haven't tested it yet.